packages/mdc-fab/_mixins.scss

```diff
@@ -25,11 +25,12 @@
 @import "@material/feature-targeting/mixins";
 @import "@material/ripple/mixins";
 @import "@material/ripple/variables";
-@import "@material/theme/functions";
-@import "@material/theme/mixins";
+@import "@material/rtl/mixins";
 @import "@material/shape/mixins";
 @import "@material/shape/functions";
-@import "@material/rtl/mixins";
+@import "@material/theme/functions";
+@import "@material/theme/mixins";
+@import "@material/touch-target/mixins";
```

Comment:
FAB's package.json missing dependency for touch target which might be causing failures in [canary release](https://github.com/material-components/material-components-web/commit/0c4d8f3923f9a089132ed8dca4062b72d3576aca/checks?check_suite_id=299135665)?

Reply:
Sent https://github.com/material-components/material-components-web/pull/5236 to fix.